tests/e2e/cht/pregnancy.spec.js

```diff
@@ -0,0 +1,124 @@
+const utils = require('../../utils');
+const commonElements = require('../../page-objects/common/common.po');
+const loginPage = require('../../page-objects/login/login.po');
+const contactsPage = require('../../page-objects/contacts/contacts.po');
+const helper = require('../../helper');
+const analyticsPo = require('../../page-objects/analytics/analytics.po');
+const moment = require('moment');
+const Factory = require('rosie').Factory;
+const formFiller = require('../../form-filling').fillForm;
+require('../../factories/reports/cht/pregnancy.js');
+require('../../factories/reports/cht/pregnancy-visit');
+require('../../factories/reports/cht/delivery');
+require('../../factories/users/cht/offline');
+require('../../factories/contacts/cht/district-hospital');
+require('../../factories/contacts/cht/clinic');
+require('../../factories/contacts/cht/woman');
```

Comment:
Yeah, otherwise `Error: The "offlineUser" factory is not defined.` is generated. I was thinking about whether or not to have something that loads them all but then you get extra which may not matter.

Reply:
> Yeah, otherwise `Error: The "offlineUser" factory is not defined.` is generated. I was thinking about whether or not to have something that loads them all but then you get extra which may not matter.

I think at some point we'll want them all to just be loaded for the suite. For now this is fine though and we can see how far we want to go with these.... my hope is far, and to have the need to load them for the whole suite :)